### PR TITLE
Deprecate Dell Omnia module and example blueprint

### PR DESCRIPTION
**Pull request by tpdownes**
The integration test removed by this PR has been removed already.

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ md_toc github examples/README.md | sed -e "s/\s-\s/ * /"
   * [hpc-slurm-sharedvpc.yaml](#hpc-slurm-sharedvpcyaml-) ![community-badge]
   * [client-google-cloud-storage.yaml](#client-google-cloud-storageyaml--) ![community-badge] ![experimental-badge]
   * [hpc-slurm-gromacs.yaml](#hpc-slurm-gromacsyaml--) ![community-badge] ![experimental-badge]
-  * [omnia-cluster.yaml](#omnia-clusteryaml--) ![community-badge] ![experimental-badge]
+  * [omnia-cluster.yaml](#omnia-clusteryaml--) ![community-badge] ![experimental-badge] ![deprecated-badge]
```
Comment (nick-stroud) [inline examples/README.md:31]: nit: I would move it to the bottom of the list. That is where we previously had the deprecated v4 examples.

Comment (nick-stroud) [inline examples/README.md:31]: I think the anchor link will now be broken. I think it wants one `-` for every badge.

**Review by nick-stroud — APPROVED** [review]

File: examples/README.md (hunk 1 above)
Comment (tpdownes) [reply examples/README.md]: Done

Comment (tpdownes) [reply examples/README.md]: That explains the mystery of the dashes. I had noticed it but blamed it on poor rendering in GitHub PR UI. Fixed.

**Pull request merged**